.azure-pipelines.yml

```diff
@@ -121,3 +122,29 @@ jobs:
   - template: tools/ci/azure/tox_pytest.yml
     parameters:
       directory: tools/wpt/
+
+- job: all_macOS
+  displayName: 'all tests (Safari Technology Preview)'
+  condition: eq(variables['Build.SourceBranch'], 'refs/heads/azure-daily-safari')
+  strategy:
+    parallel: 4 # chosen to make runtime <2h
+  timeoutInMinutes: 360
+  pool:
+    vmImage: 'macOS-10.13'
+  steps:
+  - template: tools/ci/azure/checkout.yml
+  - template: tools/ci/azure/pip_install.yml
+    parameters:
+      packages: virtualenv
+  - template: tools/ci/azure/install_fonts.yml
+  - template: tools/ci/azure/install_certs.yml
+  - template: tools/ci/azure/install_safari.yml
+  - template: tools/ci/azure/update_hosts.yml
+  - template: tools/ci/azure/update_manifest.yml
+  - script: no_proxy='*' ./wpt run --no-manifest-update --no-restart-on-unexpected --no-fail-on-unexpected --this-chunk=$(System.JobPositionInPhase) --total-chunks=$(System.TotalJobsInPhase) --chunk-type hash --log-wptreport $(Build.ArtifactStagingDirectory)/wpt_report_$(System.JobPositionInPhase).json --channel preview safari
```

Comment:
Note that I'm using `--chunk-type hash` here for a more even time between chunks. However, that does split directories across chunks. I'd like to see if there are differences to our current results and perhaps this will turn out to be a mistake.

Reply:
It was quite uneven in my testing with 8 chunks, maybe the slowest is 2x the fastest. I also expect to see discrepancies, but am curious to see how many they are. We'll probably remove this bit unless it's not as bad as suspected.